code/kitex_sample/handler/handler.go

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2024 CloudWeGo Authors
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package handler
+
+import (
+	"context"
+	"time"
+
+	"github.com/kitex-contrib/codec-dubbo-tests/code/kitex_sample/kitex_gen/hello"
+)
+
+// GreetServiceImpl implements the last service interface defined in the IDL.
+type GreetServiceImpl struct{}
+
+// Greet implements the GreetServiceImpl interface.
+func (s *GreetServiceImpl) Greet(ctx context.Context, req string) (resp string, err error) {
+	time.Sleep(time.Millisecond * 10)
```

Comment:
The reason for sleep?

Reply:
the code is used for test, i will delete it.

Reply:
Fixed